### ISSUE TITLE
listbox incorrectly renderes reordered last column

### ISSUE THREAD

**Issue by beru** [cnjinhao/nana#20]
Initial screen after the startup of program.
![image](https://cloud.githubusercontent.com/assets/1131125/6736866/d4e801fa-ceaa-11e4-99f3-d100547c1a10.png)

Top row selected.
![image](https://cloud.githubusercontent.com/assets/1131125/6736898/30152198-ceab-11e4-8e04-819dae26f2e1.png)

Last column reoreded to left-most, still correct.
![image](https://cloud.githubusercontent.com/assets/1131125/6736945/80c213bc-ceab-11e4-889f-371f24762702.png)

Column widths shrinked by dragging header borders. still looking alright.
![image](https://cloud.githubusercontent.com/assets/1131125/6737136/1d0079ca-cead-11e4-9a8c-d5aab0cc9c38.png)
(sometimes column reorederd unexpectedly after dragend timing if I dragged the column border excessively but that is another matter...)

Bottom row selected.
![image](https://cloud.githubusercontent.com/assets/1131125/6737154/3b750ace-cead-11e4-8270-f7cf3129e3eb.png)
Please pay attension to the bottom row since middle and last columns rendered improperly.

The problem disappears after I expanded ex-last-column.
![image](https://cloud.githubusercontent.com/assets/1131125/6737230/a5f6d01c-cead-11e4-936e-eb4f2c4fd0bc.png)

**Comment by beru** [comment]
```c++
#include <nana/gui.hpp>
#include <nana/gui/widgets/listbox.hpp>

using namespace nana;

int main()
{
	using namespace nana;
	form fm(API::make_center(500, 200));
	fm.caption(L"listbox test");
	
	listbox lst(fm, {0, 0, 500, 200});
	lst.append_header(L"Name");
	lst.append_header(L"Gender");
	lst.append_header(L"Age");
	auto cat = lst.at(0);
	cat.append({L"齐天大圣", L"男性", L"2418"});
	cat.append({L"唐三藏", L"男性", L""});
	cat.append({L"猪八戒", L"男性", L""});
	cat.append({L"沙和尚", L"男性", L""});
	cat.append({L"白龍馬", L"种马", L"0x7ff0000000000000"});

	fm.show();
	exec();
}
```

**Comment by qPCR4vir** [comment]
Fixed in #21

**Comment by cnjinhao** [comment]
Thanks you both! This fix has been merged into the repo.
